[2025-05-16T18:47:35.004+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-16T18:47:31.776515+00:00/task_id=transform_api permission to 509
[2025-05-16T18:47:35.004+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-16T18:47:31.776515+00:00/task_id=transform_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-16T18:47:31.776515+00:00/task_id=transform_api'
[2025-05-16T18:47:35.027+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-16T18:47:31.776515+00:00/task_id=transform_api permission to 509
[2025-05-16T18:47:35.028+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-16T18:47:31.776515+00:00/task_id=transform_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-16T18:47:31.776515+00:00/task_id=transform_api'
[2025-05-16T18:47:35.062+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dimensional_model.transform_api manual__2025-05-16T18:47:31.776515+00:00 [queued]>
[2025-05-16T18:47:35.069+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dimensional_model.transform_api manual__2025-05-16T18:47:31.776515+00:00 [queued]>
[2025-05-16T18:47:35.069+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2025-05-16T18:47:35.081+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): transform_api> on 2025-05-16 18:47:31.776515+00:00
[2025-05-16T18:47:35.088+0000] {standard_task_runner.py:60} INFO - Started process 215 to run task
[2025-05-16T18:47:35.091+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dimensional_model', 'transform_api', 'manual__2025-05-16T18:47:31.776515+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/etl_load_postgres_dag.py', '--cfg-path', '/tmp/tmp7lfz2y4a']
[2025-05-16T18:47:35.092+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask transform_api
[2025-05-16T18:47:35.132+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-16T18:47:31.776515+00:00/task_id=transform_api permission to 509
[2025-05-16T18:47:35.132+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-16T18:47:31.776515+00:00/task_id=transform_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-16T18:47:31.776515+00:00/task_id=transform_api'
[2025-05-16T18:47:35.134+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_dimensional_model.transform_api manual__2025-05-16T18:47:31.776515+00:00 [running]> on host e3308f139649
[2025-05-16T18:47:35.318+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_dimensional_model' AIRFLOW_CTX_TASK_ID='transform_api' AIRFLOW_CTX_EXECUTION_DATE='2025-05-16T18:47:31.776515+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-16T18:47:31.776515+00:00'
[2025-05-16T18:47:35.338+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-16T18:47:35.347+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_dimensional_model, task_id=transform_api, execution_date=20250516T184731, start_date=20250516T184735, end_date=20250516T184735
[2025-05-16T18:47:35.384+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-16T18:47:35.402+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
